mne/tests/test_epochs.py

```diff
@@ -1254,5 +1254,34 @@ def test_array_epochs():
     assert_equal(epochs.drop_log[0], ['EEG 006'])
     assert_equal(len(events), len(epochs.selection))
 
+def test_dataframe_time_indexing():
+    '''Make sure that conversion to dataframe
+    doesn't duplicate time indices due to rounding
+    error + floating point precision'''
+    # Synthesize data
+    data = np.random.randn(2, 10000)
+    data_event = np.zeros(data.shape[1])
+    data_event[[2, 2000]] = 1  # Insert some events
+    data_raw = np.vstack([data, data_event])
+
+    # For the Info object
+    ch_types = ['eeg']*data_raw.shape[0]
+    ch_names = ['{0}'.format(i) for i in range(data_raw.shape[0])]
+    ch_names[-1] = 'stim'
+    ch_types[-1] = 'misc'
+
+
+    # Create base MNE objects
+    info = create_info(ch_names, 1000, ch_types=ch_types)
+    raw = io.RawArray(data_raw, info)
+    events = find_events(raw, 'stim')
+    epochs = Epochs(raw, events, 1, -.2, 1.5)
+
+    # Convert to dataframe
+    epochs_df = epochs.as_data_frame()
+    epochs_times = epochs_df.index.get_level_values('time')
+
+    assert epochs_times.duplicated().values.sum() == 0, 'Conversion to DataFrame duplicated timepoints'
```

Comment:
use `assert_true(...)` or even better `assert_equal`, we use `nose` not `py.test` standards around here :)